snippets/typescript.json

```diff
@@ -160,5 +160,19 @@
       "export class ${Name}Module { }",
       "$0"
     ]
+  },
+  "Angular Directive": {
+    "prefix": "a-directive",
+    "description": "Angular directive snippet",
+    "body": [
+      "import { Directive, ElementRef } from '@angular/core';",
+      "",
+      "@Directive({ selector: '[${feature}]' })",
+      "export class ${Feature}Directive {",
+      "\tconstructor(el: ElementRef) {",
```

Comment:
lets avoid the import since the ctor argument is now optional. we wont know what they want to add.

and in the future, the tooling with TS should make this easier.